baseapp/baseapp.go

```diff
@@ -567,6 +567,13 @@ func getState(app *BaseApp, mode runTxMode) *state {
 	return app.deliverState
 }
 
+func (app *BaseApp) initializeContext(ctx sdk.Context, mode runTxMode) sdk.Context {
+	if mode != runTxModeSimulate {
```

Comment:
(nit) Writing this with an `if mode == runTxModeSimulate` is a bit more obvious (that we're doing something special for simulation).

Reply:
Incorporated, thanks.